kolibri/core/content/utils/paths.py

```diff
@@ -277,7 +277,7 @@ def zip_content_path_prefix():
 
     if path_prefix != "/":
         path_prefix = "/" + path_prefix
-    return path_prefix
+    return get_content_url(path_prefix)
```

Comment:
i have this error : 
Uncaught (in promise) Error: Ajax error for /content/storage/6/b/6b843f1b8e7e37300fc3bce3dfa81706.h5p : 404 Not Found

and

Validation failed for 'download_url' in  
Object { id: Getter & Setter, contentnode: Getter & Setter, priority: Getter & Setter, preset: Getter & Setter, supplementary: Getter & Setter, thumbnail: Getter & Setter, checksum: Getter & Setter, available: Getter & Setter, file_size: Getter & Setter, extension: Getter & Setter, … }
mixin.js:61

Reply:
I've not seen the first issue - and that's precisely where the H5P file should be, so that's a little odd.

The second is a residual prop on the renderer mixin that is no longer being passed in, nothing to do with this work.

Reply:
this is what i see:

in a fresh installation, i have the first error to not load H5P in front-end, however the file missing is actually in content.

However, when i use      "return path_prefix" i can load h5p normaly, but when i switch to new "return get_content_url(path_prefix)" the H5P load and no error popup. so maybe it need a fresh content (.kolibri) to see error or maybe something i do wrong

Reply:
Thanks @MBKayro - the change I made was too aggressive for the result I was looking for, have updated it.